ai_engine_sdk/client.py

```diff
@@ -335,6 +343,67 @@ async def delete_function_group(self, function_group_id: str):
             endpoint=f"/v1beta1/function-groups/{function_group_id}/",
         )
         logger.debug(f"Function group deleted: {function_group_id}")
+        raw_response: dict = await make_api_request(
+            api_base_url=self._api_base_url,
+            api_key=self._api_key,
+            method='GET',
+            endpoint="/v1beta1/function-groups/public/"
```

Comment:
do we still have the /public? wasn't that replaced?

Reply:
Still existing, but just because we did not want break anything.
But yeah, definitely it will be good to start to remove dependencies on it.

The only thing we should take into account is: Is the update in the "/function-group" endpoint deployed in production already? (the sdk is meant, in general, to work with production)

Reply:
it is not yet. but we don't do sdk release before prod release. if its compatible with current staging lets keep it and remove as next step